members-list.md

```diff
@@ -18,6 +18,11 @@ Also don't be a cop-out by saying something like "I am happy/looking forward to
 ## The list 
 [Add your details here]`<--Don't remove this line`<br/>
 
+**Name**:  Avnish Aryan Jha <br/>
+**SRN**:  PES1UG19EC073 <br/>
+**Link to Profile**:  [avnish100](https://github.com/name)  </br>
+**About Me**: I'm interested in varied fields from data science to psychology.Enjoy learning and creating.<br/>
```

Comment:
Hi! Welcome!
Could you link your GitHub profile, `github.com/name` is supposed to be a placeholder and `name` should be replaced by your GitHub handle :)

Reply:
okay so should i create another pull request or can i edit this one somehow